go-sample/playground/index.html

```diff
@@ -652,6 +671,13 @@
     font-weight: 600;
   }
 
+  p.curl_command {
```

Comment:
Can probably skip the element type selector:

```suggestion
  .curl_command {
```

Reply:
This actually did break the styling. I'm not that good at CSS, so I couldn't tell you why, but without this it seemed like the style for the table heading took precedence over the style for the curl command. I ended up just reverting this change.